Objective-C/Tests/VectorSearchTest.m

```diff
@@ -6,6 +6,7 @@
 //  COUCHBASE CONFIDENTIAL -- part of Couchbase Lite Enterprise Edition
 //
 
+#import "CBLErrors.h"
```

Comment:
This is already included in CouchbaseLite.h which is included in the CBLTestCase.h so this is not needed.